pmultiqc/modules/quantms/maxquant.py

```
@@ -442,6 +443,112 @@ def get_evidence(file_path):
+    # 4. Missed Cleavages
+    missed_cleavages = {key: value["0"] / 100 for key, value in msms_missed_cleavages.items()}
+
+    # 5. Missed Cleavages Var
+    mc_median = np.median(list(missed_cleavages.values()))
+    missed_cleavages_var = dict(
+        zip(
+            missed_cleavages.keys(),
+            list(map(lambda v: 1 - np.abs(v - mc_median), missed_cleavages.values())),
+        )
+    )
+    for raw_file in missed_cleavages.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)